azkaban-web-server/src/main/java/cloudflow/servlets/ExecutionServlet.java

```diff
@@ -132,9 +122,95 @@ protected void handleGet(HttpServletRequest req, HttpServletResponse resp, Sessi
     return map;
   }
 
+  @Override
+  protected void handleGet(HttpServletRequest req, HttpServletResponse resp, Session session)
+      throws IOException {
+    Map<String, String> templateVariableMap = new HashMap<>();
+    String requestUri = req.getRequestURI();
+
+    if (GET_ALL_EXECUTIONS_URI.match(requestUri, templateVariableMap)) {
+      handleAllExecutions(req, resp);
+    } else if (GET_SINGLE_EXECUTION_URI.match(requestUri, templateVariableMap)) {
+      handleSingleExecution(req, resp);
+    } else if (GET_JOB_EXECUTION_INFO_URI.match(requestUri, templateVariableMap)) {
+      handleExecutionWithJobInfo(req, resp, session, templateVariableMap);
+    } else {
+      sendErrorResponse(resp, HttpServletResponse.SC_NOT_FOUND, DEFAULT_404_ERROR_MESSAGE);
+    }
+  }
+
```

Comment:
Most of this was part of Shardool's change. You might have to pull and merge.

Reply:
that's done, this change is only a code movement.